packages/react/src/components/Slider/Slider.types.ts

```diff
@@ -225,3 +225,64 @@ export interface ISliderStyles {
    */
   zeroTick: IStyle;
 }
+
+export interface IClassNameProp {
```

Comment:
Thanks for doing this, but this is actually not quite what I meant, sorry (I should have been more specific). What I meant was inline on the individual props objects, declare them to have types like `React.HTMLAttributes<HTMLDivElement>`. That way it will automatically validate that anything being added is a valid HTML element prop.

Reply:
Either that or prop types for Fluent UI React elements if that's what's used for the slot currently (the slots would ideally have more generic props later)

Reply:
LoL I see what you are saying now. Updated. Please take another look